_posts/2018-05-17-self-organization.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Self Organization"
+date: 2018-05-17 12:0:00 +0200
+categories: motivation how-we-learn
+author_name: Ugurcan Sengit
+author_url : /author/ugurcansengit
+author_avatar: ugurcansengit
+read_time : 8
+feature_image: posts/2018-04-20-holidaycheck-apprenticeship/apprenticeship.png
+---
+
+I remember the very first day as an apprentice at [HolidayCheck](https://twitter.com/holidaychecklab). Honestly, it's only four months ago, not so hard to recall. The thing is, it feels like it's been a lifetime already. All the things I've learnt during the apprenticeship... One just can't put a price tag on them. Time management and self-organization made them happen.
+
+At the very first day, I was both excited and afraid. Because I've never had six months of free space to improve myself after starting my professional career. The first challenge for me was to shape these six months. I've needed to create a structure and follow it strictly in order not to get lost during my journey. After my apprenticeship half-time presentation, I think I've nailed it, that's why I write this blog post to share my process with you.
+
+In my [3rd blog post](https://www.sengitu.com/posts/no-estimates/), I was so happy to announce my plan. After having that plan, the best thing I've done was to stick with it. I've shaped my plan over time to overcome problems I had, but I've never got out of the frame. Currently, in my [Trello board](https://trello.com/b/KlLdup7o/ugurcan-sengit-apprenticeship-board), I've following columns;
```

Comment:
At the very end, you're missing a "the". "I have the following columns". Also, consider using the long form "I have" instead of "I've". Don't know why but it sounds better in my mind. Feel free to keep it any way you like though.